src/client/actions/GuildMemberUpdate.js

```diff
@@ -23,6 +15,7 @@ class GuildMemberUpdateAction extends Action {
         /**
          * Emitted whenever a guild member changes - i.e. new role, removed role, nickname.
          * Also emitted when the user's details (e.g. username) change.
+         * Even when emitted from a user detail change, the user object will be identical.
```

Comment:
could there not be some logic to have these not identical?

Reply:
> A consideration was placed on making the user object represent the change, however, given all guilds that the bot shares with the member being updated will emit this event, it would become inconsistent at scale.

There could be logic, however, this would only work for the first guildMemberUpdate / presenceUpdate for any given change to a user. Thus a majority of events would still end up having this issue and it becomes inconsistent.